python_modules/dagster/dagster/daemon/daemon.py

```diff
@@ -22,7 +23,8 @@ def get_default_daemon_logger(daemon_name):
 
 
 DAEMON_HEARTBEAT_ERROR_LIMIT = 5  # Show at most 5 errors
-TELEMETRY_LOGGING_INTERVAL = 3600  # Interval (in seconds) at which to log that daemon is alive
+TELEMETRY_LOGGING_INTERVAL = 3600 * 24  # Interval (in seconds) at which to log that daemon is alive
+TELEMETRY_DAEMON_SESSION_ID = str(uuid.uuid4())
```

Comment:
nit i associate this all-caps format with constants, which is not quite what this is